mediaire_toolbox/queue/redis_wq.py

```diff
@@ -94,16 +97,88 @@ def _lease_exists(self, item):
     def put(self, item):
         self._db.lpush(self._main_q_key, item)
 
+    @staticmethod
+    def _get_timestamp(timeunit):
+        if timeunit == 'sec':
+            return time.gmtime(time.time()).tm_sec
+        if timeunit == 'min':
+            return time.gmtime(time.time()).tm_min
+        if timeunit == 'hour':
+            return time.gmtime(time.time()).tm_hour
+        raise ValueError('Invalid timeunit {}'.format(timeunit))
+
+    @staticmethod
+    def _get_limit_expirytime(timeunit):
+        if timeunit == 'sec':
```

Comment:
let's comment on what this method does (that it always returns seconds, and why do we need them)